lib/datadog/core/environment/ext.rb

```diff
@@ -4,6 +4,7 @@
 module Datadog
   module Core
     module Environment
+      # @public_api
```

Comment:
A bunch of these modules aren't actually showing up on the sidebar:

<img width="256" alt="Screen Shot 2021-12-14 at 19 00 26" src="https://user-images.githubusercontent.com/2785847/146054119-6f0ae878-0200-4fab-b007-7f8e730d2e1f.png">

-- is this deliberate?

Reply:
Addresses this!